Toggl.Core.UI/ViewModels/SuggestionsViewModel.cs

```diff
@@ -59,22 +58,17 @@ public override async Task Initialize()
             Suggestions = interactorFactory.ObserveWorkspaceOrTimeEntriesChanges().Execute()
                 .StartWith(Unit.Default)
                 .SelectMany(_ => getSuggestions())
-                .AsDriver(onErrorJustReturn: new Suggestion[0], schedulerProvider: schedulerProvider);
+                .AsDriver(onErrorJustReturn: ImmutableList.Create<Suggestion>(), schedulerProvider: schedulerProvider);
 
             IsEmpty = Suggestions
-                .Select(suggestions => suggestions.Length == 0)
+                .Select(suggestions => suggestions.Count() == 0)
```

Comment:
There is a property `Count` which should be used instead of the property instead of the LINQ extension method. But additionally we have just a right ext. method for this.

```suggestion
                .Select(suggestions => suggestions.None())
```